docs/plat/nxp-layerscape.rst

```diff
@@ -0,0 +1,175 @@
+NXP SoCs - Overview
```

Comment:
The file has to be rst formatted as it will enable auto-generation of HTML and other formats easily. This will enable easier migration to trustedfirmware.org later.